instrumentation/logback/logback-mdc-1.0/javaagent/README.md

```diff
@@ -1,5 +1,15 @@
 # Settings for the Logback MDC instrumentation
 
-| System property | Type | Default | Description |
-|---|---|---|---|
-| `otel.instrumentation.logback-mdc.add-baggage` | Boolean  | `false` | Enable exposing baggage attributes through MDC. |
+You can enable/disable this feature by set system properties or environment variables.
+
+For system properties
+
+| Name | Type | Default | Description |
+|-|-|-|-|
+| `otel.instrumentation.logback-mdc.add-baggage` | Boolean | `false` | Enable exposing baggage attributes through MDC. |
+
+For environment variables
+
+| Name | Type | Default | Description |
+|-|-|-|-|
+| `OTEL_INSTRUMENTATION_LOGBACK_MDC_ADD_BAGGAGE` | Boolean | `false` | Enable exposing baggage attributes through MDC. |
```

Comment:
In the case of javaagent instrumentations the env vars are already sort of implicitly [documented](https://opentelemetry.io/docs/instrumentation/java/automatic/agent-config/#configuring-with-environment-variables), and we don't really want to duplicate that here.

```suggestion
```

I know that you probably added it here because of my other comment in the `library-autoconfigure` module, but that situation is a bit different -- it is **very** rare for us to have any non-programmatic configuration in library modules, so in that particular case a bit of extra doc is quite helpful.
Sorry for the confusion.

Reply:
Okay.